src/Compilers/VisualBasic/Test/Semantic/Semantics/BinaryOperators.vb

```diff
@@ -1502,6 +1502,25 @@ End Module
             CompileAndVerify(compilation, expectedOutput:="58430604")
         End Sub
 
+        <Fact>
+        Public Sub UncheckedExpression()
+
+            Dim compilationDef =
+<compilation>
+    <file name="a.vb">
+Module M
+    Sub Main()
+        Dim ten = 10
+        Dim i2 = Unchecked ( 2147483647  + ten )
```

Comment:
Integer.MaxValue would make intent clearer

Reply:
The other test that I didn't see is
```
Dim i3 as ULong= UnChecked (ULong.MaxValue + 1)
``` 
which today gives an Error BC30439 Constant expression not representable in type 'ULong'